one_fm/overrides/leave_application.py

```diff
@@ -424,7 +424,8 @@ def on_update(self):
         if self.status == "Approved":
             if getdate(self.from_date) <= getdate() <= getdate(self.to_date):
                 # frappe.db.set_value(), will not call the validate.
-                frappe.db.set_value("Employee", self.employee, "status", "Vacation")
+                if self.leave_type !='Sick Leave':
```

Comment:
@Talleyrand333 What about other leave types other than Sick Leave? For Example Casual leave, Privilege leave e.t.c